app/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "dev": "NODE_OPTIONS='--inspect' next dev",
+    "dev": "nodemon --watch backend server",
```

Comment:
This adds nodemon so the backend reboots if we change it but removes debugger support by removing the `NODE_OPTIONS`.